README.md

```diff
@@ -49,8 +49,8 @@ clientSettings:
 ```
 
 * `clientSettings.concurrency` is an optional parameter that shows the concurrency of Nebula Graph Client, i.e. the connection number of Nebula Graph Server, the default value is 10.
-* `clientSettings.channelBufferSize` is an optional parameter that shows the buffer size of cache queue for each Nebula Graph Client, the default value is 128.
-* `clientSettings.space` is a **required** parameter that specifies which `space` the data will be importing into. Do not import data to multiple spaces at one time for the performance sake.
```

Comment:
`clientSettings.space` is a **required** parameter that specifies which `space` the data will be imported to. Do not import data to multiple spaces at one time for performance sake.